pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -184,6 +184,44 @@ public void testQuotedStrings() {
             .getIdentifier().getName(), "Martha''s Vineyard");
   }
 
+  @Test
+  public void testExtract() {
+    try {
+      //@formatter:off
```

Comment:
I don't think we need to turn off/on the formatter since there is no special formatting needed